lisa/schema.py

```diff
@@ -381,6 +381,19 @@ class NodeSpace(search_space.RequirementMixin, TypedSchema, ExtendableSchemaMixi
         default=search_space.IntRange(min=0),
         metadata=metadata(decoder=search_space.decode_count_space),
     )
+    data_disk_caching: str = field(
+        default=constants.DATADISK_CACHING_TYPE_NONE,
+        metadata=metadata(
+            validate=validate.OneOf(
+                [
+                    constants.DATADISK_CACHING_TYPE_NONE,
+                    constants.DATADISK_CACHING_TYPE_READONLY,
+                    constants.DATADISK_CACHING_TYPE_READYWRITE,
+                ]
+            ),
+        ),
+    )
+    data_disk_size_type: str = "S4"
```

Comment:
As we discussed, it should be IOPS, not disk size type. It's not translatable to other platforms. The type should be a IntRange

Reply:
fix